Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -7,9 +10,15 @@ public static class ArrayOperations
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
+        /// 
+        private static bool IsArrayNullOrEmpty(int[] array)
```

Comment:
I love the private function! Though please do put it all the way down. Low level code should be down